src/validators/restrictionKey.validator.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ *
+ * Copyright 2018-present NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import { ExpectedError, ValidationContext, Validator } from 'clime';
+
+export class MosaicRestrictionKeyValidator implements Validator<string> {
+    validate(value: string, context?: ValidationContext): void {
+        const regExp = /^[a-zA-Z]+[a-zA-Z0-9_]+?$/g;
```

Comment:
MosaicRestrictionKey should be a UInt64

Reply:
@dgarcia360 

The function `MosaicRestrictionKeyValidator` is uesed with string type parameter, because it will check the parameter passing from the command. We could converse it into UInt64 after checking. Here, I just want to limit the format of a key. 
Did you mean limiting the command parameter as an UInt64 variable? 
For example: 
```ts
@option({
  flag: 't',
  description: 'Test.',
  validator: new MosaicRestrictionKeyValidator(),
})
test: UInt64;
```

Reply:
No, ``key`` option is still read as a string (key: string;)  but the validator should test that this string is a valid UInt64.

For example, you could do:

`` try { new UInt64(value) } catch { throw new ExpectedError('Invalid restriction key');}``